bonding.py

```diff
@@ -260,7 +260,7 @@ def defaults(prompt, default):
         sys.exit(0)
 
 
-def peers(quiet=True):
+def peers(quiet=True, wait=5):
```

Comment:
for consistency sake, lets not make this too generic sounding, and go with calling the argument `peerswait`